README.md

```diff
@@ -4,3 +4,5 @@ HTMLslyde
 I randomly thought about setting multiple HTML divs with 100% height, and it turns out this is a really easy way to have vertical scrolling "slides".
 
 I want to implement some cleaner transitions as seen on the login/registration forms on the demo  [here](http://tympanus.net/codrops/2012/03/27/login-and-registration-form-with-html5-and-css3/)
+
+This project uses no JavaScript. 
```

Comment:
Let's move this sentence  up so it's the second "paragraph".
I'd prefer to say "The intent of this project is to create an HTML slideshow without using JavaScript."